zerver/tests/test_subs.py

```diff
@@ -7764,7 +7764,7 @@ def test_gather_subscriptions(self) -> None:
             polonius.id,
         ]
 
-        with self.assert_database_query_count(50):
+        with self.assert_database_query_count(53):
```

Comment:
Why does this add 3 queries? If it's because we're sending 3 DMs, and we've added one query for each, that's OK.

Reply:
Yes, exactly. The `add_subscriptions_backend` [view](https://github.com/zulip/zulip/blob/d978363a75d7c3b36e051df6aba21984d505a8ac/zerver/views/streams.py#L596) calls the `send_messages_for_new_subscribers` [function](https://github.com/zulip/zulip/blob/d978363a75d7c3b36e051df6aba21984d505a8ac/zerver/views/streams.py#L769) at the end to send 3 DMs to the [3 invitees](https://github.com/zulip/zulip/blob/f9800ac215fd6e89b39fa4f44409ad9819bb8aab/zerver/tests/test_subs.py#L7762-L7764).

I also printed the before/after queries; three new queries are on the huddle table:
[queries-after.txt](https://github.com/user-attachments/files/19640998/queries-after.txt)
[queries-before.txt](https://github.com/user-attachments/files/19640999/queries-before.txt)